internal/compiler-interface/src/main/java/xsbti/compile/DefinesClass.java

```diff
@@ -8,14 +8,19 @@
 package xsbti.compile;
 
 /**
- * Determines if a classpath entry contains a class.
+ * Determine whether a classpath entry contains a class.
  *
- * The corresponding classpath entry is not exposed by this interface. It's tied
- * to it by a specific class that implements this interface.
+ * The corresponding classpath entry is not exposed by this interface.
  */
 public interface DefinesClass {
     /**
-     * Returns true if the classpath entry contains the requested class.
+     * Return true if the classpath entry contains the requested class.
```

Comment:
Docs should be in the 3rd person.

So "DefinesClass determines whether..." and "DefinesClass.apply returns true if...". Not "determine" and "return".